techlibs/gowin/synth_gowin.cc

```diff
@@ -94,13 +94,16 @@ struct SynthGowinPass : public ScriptPass
 		log("		  The following families are supported:\n");
 		log("        'gw1n', 'gw2a', 'gw5a'.\n");
 		log("\n");
+		log("    -setundef\n");
+		log("        run 'setundef' with '-undriven -params -zero' options\n");
```

Comment:
not so sure this help comment is useful; perhaps something like `set undriven wires and parameters to zero`.